protocol/communities/manager.go

```diff
@@ -2816,6 +2833,15 @@ func (m *Manager) HandleCommunityRequestToJoin(signer *ecdsa.PublicKey, receiver
 			}
 		}
 
+		// Check if we reached the limit, if we did, change the community setting to be On Request
+		if community.AutoAccept() && community.MembersCount() >= maxNbMembers {
```

Comment:
Do we want to restore `PermissionAccess` to original value when the there's less nu members again? 🤔

Reply:
That would be a good idea, but I'll keep it simple for now. In theory, if we do reach the limit and the app still behaves nicely, we would create a new build with a higher limit first.